packages/langium/src/grammar/lsp/grammar-semantic-tokens.ts

```diff
@@ -54,3 +57,43 @@ export class LangiumGrammarSemanticTokenProvider extends AbstractSemanticTokenPr
     }
 
 }
+
+export interface DecodedSemanticToken {
```

Comment:
Suggestion: I believe that this interface (and the class below) belongs in the `semantic-tokens.ts` file, not the `grammar-semantic-tokens.ts` file.

Reply:
Do you mean the `semantic-token-provider.ts`? My branch has no `semantic-tokens.ts`

Reply:
Done